config/database.yml.sample

```diff
@@ -16,4 +16,4 @@ test:
 
 production:
   <<: *default
-  detabase: rails_project_production
+  detabase: rails_project_production
```

Comment:
do not commit this one

Reply:
ma'am kaya ko po cinommit kasi dinelete ko po siya kahapon, paano po siya maccheck ng github?